### PR TITLE
Micro MLA

### PR DESCRIPTION
**Pull request by MHDante**

### CODE REVIEW

File: com.unity.ml-agents/Runtime/Actuators/ActionSegment.cs [hunk 1]
```diff
@@ -1,10 +1,10 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
-using System.Diagnostics;
 using System.Linq;
+using UnityEngine;
```
Comment (CubeMD) [inline com.unity.ml-agents/Runtime/Actuators/ActionSegment.cs:5]: Dont need

File: com.unity.ml-agents/Runtime/Actuators/IActionReceiver.cs [hunk 2]
```diff
@@ -2,7 +2,7 @@
 using System.Linq;
 using UnityEngine;
 
-namespace Unity.MLAgents.Actuators
+namespace TransformsAI.MicroMLAgents.Actuators
```
Comment (CubeMD) [inline com.unity.ml-agents/Runtime/Actuators/IActionReceiver.cs:5]: Mb change the name of the file?

**Review by CubeMD — APPROVED** [review]

File: com.unity.ml-agents/Runtime/Actuators/ActionSegment.cs (hunk 1 above)
Comment (MHDante) [reply com.unity.ml-agents/Runtime/Actuators/ActionSegment.cs:5]: will fix.

File: com.unity.ml-agents/Runtime/Actuators/IActionReceiver.cs (hunk 2 above)
Comment (MHDante) [reply com.unity.ml-agents/Runtime/Actuators/IActionReceiver.cs:5]: kept for diff purposes.

**Pull request merged**